src/ad/nrt.ts

```diff
@@ -58,7 +58,7 @@ const impl: Validator = {
 
     const mid = v.substr(1, v.length - 2);
 
-    if (!strings.isalpha(v[0]) || !strings.isalpha(v[v.length - 1])) {
+    if (!strings.isalphanumeric(v[0]) || !strings.isalphanumeric(v[v.length - 1])) {
```

Comment:
Re-reading the NRT specification it's `A-NNNNNN-A`, so the validators should be `strings.isalpha()` in both cases.  Unless you have an alternate reference.